tests/acceptance/EnableAndDisableSignupConfirmationCest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use Codeception\Util\Locator;
+use Facebook\WebDriver\WebDriverKeys;
+use MailPoet\Test\DataFactories\Form;
+
+require_once __DIR__ . '/../DataFactories/Form.php';
+
+class EnableAndDisableSignupConfirmationCest {
```

Comment:
Could you sort the methods here? It is not very obvious what is a test and what is a helper function. We should add tests on top and helpers on the bottom. It is quite a common practice. If someone opens a file and starts reading it should go from the public stuff.